packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -44,6 +45,11 @@ class CupertinoPageScaffold extends StatelessWidget {
   /// [navigationBar].
   final Widget child;
 
+  /// The color of the widget that underlies the entire Scaffold.
```

Comment:
That should probably be a lower case s on "scaffold" because otherwise people might think we mean the [Scaffold] widget.